lib/src/query_service.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:js';
```

Comment:
consider putting all service related classes into their own services folder

Reply:
bump

Reply:
Thinking of doing this in a separate pr because I also want to put all js wrapper classes into their own js folder